CMakeModules/templates/FindOpendTect.cmake.in

```diff
@@ -62,15 +62,19 @@ endif()
 
 #Setup OpendTect's binary location (if not done)
 if ( NOT DEFINED OD_BINARY_BASEDIR )
-    if ( APPLE )
-	set ( OD_BINARY_BASEDIR ${OpendTect_DIR}/../.. )
-    else()
 	set ( OD_BINARY_BASEDIR ${OpendTect_DIR} )
-    endif()
 endif()
 if ( NOT ONLY_SET_VAR )
     #Include all OpendTect Utility macros
-    include( ${OpendTect_DIR}/CMakeModules/OD_SetupOD.cmake )
+    if ( EXISTS "${OpendTect_DIR}/CMakeModules/OD_SetupOD.cmake" )
+        include( "${OpendTect_DIR}/CMakeModules/OD_SetupOD.cmake" ) 
+    elseif ( EXISTS "${OpendTect_DIR}/Contents/Resources/CMakeModules/OD_SetupOD.cmake" )
+        include( "${OpendTect_DIR}/Contents/Resources/CMakeModules/OD_SetupOD.cmake" ) 
+    elseif ( EXISTS "${OD_BINARY_BASEDIR}/Contents/Resources/CMakeModules/OD_SetupOD.cmake" )
```

Comment:
The cases based on OD_BINARY_BASEDIR should be removed.
They are not needed if OD_BINARY_BASEDIR == OpendTect_DIR,
and the file should not be located in a subdir of OD_BINARY_BASEDIR if OD_BINARY_BASEDIR != OpendTect_DIR